.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.7","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["F:\\Program Files\\Java\\jdk-9.0.4/bin/java","-Xms100m","-Xmx100m","-classpath","C:\\Users\\Darwin\\AppData\\Roaming\\JetBrains\\IdeaIC2021.1\\plugins\\Scala\\launcher\\sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
This file should not be committed, contains user specific data (classpath)

Reply:
Thanks for pointing it out, we'll change it in the next PR with exercise #1.